### DISCUSSION TITLE
The response badge gives an impression that everything is fine when the site is down

### DISCUSSION THREAD

**Discussion by ganesshkumar** [upptime/upptime#253] [Q&A] [answered]
I noticed that the response time badge color varies based on the response time in milliseconds. That's a nice idea, however when the website is down, it shows 0ms in all green color. If I see only the response time badges, it looks like everything is fine. Will it be better if we check if the site the up, when the response time is 0ms, to decide if the badge should be green or red?

![image](https://user-images.githubusercontent.com/2135089/107654071-4d2c8180-6ca8-11eb-8a57-e206d562b515.png)

**Comment by AnandChowdhary** [comment, marked as answer]
This is now fixed, and the response time badge is red when the response time is exactly 0.
